{{cookiecutter.project_slug}}/backend/app/app/crud/base.py

```diff
@@ -0,0 +1,218 @@
+import logging
+from typing import List, Optional, Any
+
+from fastapi.encoders import jsonable_encoder
+from sqlalchemy.orm import Session
+
+from app.db.base_class import Base
+from pydantic import BaseModel
+
+
+class CrudBase:
+
+    def __init__(self, model: Base, schema: BaseModel):
```

Comment:
I think it would be correct here to use `Type[BaseModel]` but I'm no 100% sure.

Reply:
not sure either... I will have to take a closer look at this one (or get feedback from others ;) )